### ISSUE TITLE
Allow user-defined coloring for tags

### ISSUE THREAD

**Issue by morbus** [mcdemarco/dotgraph#12]
I'm currently mapping a story that increases a stat either a little (+), a little more (++), or a lot (+++). I tag passages as "Stat+", "Stat++", and "Stat+++". These three tags get three different colors in the default implementation. I'd like the ability to define my own colors for tags, perhaps something like `"tagColors: [ {"tagone+": "#111111"}, {"tagone++": #111111"}, ... ]"`. This would allow me to either a) color similar tags the same color [useful in combination with #11] or b) shade similar tags in the same way ("all blue shades mean this stat will be increased. got it.").

**Comment by mcdemarco** [comment]
There is also a tag-colors field in StoryData, though the values are restricted by the Twine GUI and it's unclear what would happen if you used hex values there.  Nevertheless, it would be handy to pick that up.

**Comment by morbus** [comment]
Agree. Discovered that while switching to tweego/twee3. I think that's probably the way to go here.

**Comment by mcdemarco** [comment]
The tag color list for Twine is [here](https://github.com/klembot/twinejs/blob/main/src/util/color.ts).
